src/classes/AccountDeDuper.cls

```diff
@@ -31,7 +33,7 @@ Public List<Account> SearchAccount (Account pCurrentRecord, String pAccountName,
             else {
                 ListOfMatches = [select id, Name, BillingStreet, BillingCity, BillingPostalCode, BillingState, Phone, Category__c, Owner.Alias, Record_Type_Name__c from Account where Name=:pAccountName and BillingPostalCode=:pPostCode AND RecordTypeId =:pRecordTypeId order by Name LIMIT :currentLimit ];
             }
-            System.Debug ('##################Search1: ' + ListOfMatches.size());
```

Comment:
* This method is empty, I suggest to remove it.
* Methods should use the `lowerCamelCase` naming convention. I suggest to rename accordingly.